ArmTemplates/azure-search.md

```diff
@@ -0,0 +1,22 @@
+# Azure Search
+
+Creates an Azure Search
+
+## Paramaters
+
+azureSearchName: (required) string
+
+Name of key vault. Will be created in the same resource group as the script is run and in the default location for resource group.
+
+azureSearchSku: (optional) string
+
+Can be either free, basic, standard, standard2 or standard3.
+Will default to standard if not supplied.

```

Comment:
Note:  you cannot scale from basic to standard.

Reply:
Added additional comments